README.md

```diff
@@ -107,6 +108,7 @@ API | Description | Auth | HTTPS | Link |
 ### Business
 API | Description | Auth | HTTPS | Link |
 |---|---|---|---|---|
+| Charity Search | Non-profit charity data | `apiKey` | No | [Go!](http://charityapi.orghunter.com/) |
```

Comment:
`Affiliate Basic Plan - Free for Affiliate Members.` - can I be an Affiliate Member for free?

Reply:
Yep, it's free to sign up and your key comes when you confirm your email: https://orghunter.3scale.net/